lib/connector.js

```diff
@@ -225,8 +224,26 @@ Connector.prototype.id = function(model, prop) {
  * @param {String} model The target model name
  * @param {String} prop The property name
  */
-Connector.prototype.propMapping = function(model, prop) {
-  return SQLConnector.SQLConnector(model, prop);
+Connector.prototype.propertyMapping = function(model, property) {
+  const prop = this.getPropertyDefinition(model, property);
```

Comment:
Where is this code being copied from?

Reply:
Ah, I see. I was searching for `mappingName` in the existing code. That's why I couldn't find it. :S